packages/core/src/RenderingEngine/RenderingEngine.ts

```diff
@@ -221,7 +221,11 @@ class RenderingEngine implements IRenderingEngine {
     }
 
     // 8. Resize the offScreen canvas to accommodate for the new size (after removal)
-    this.resize();
+    // Note: Resize should not reset pan and zoom in disableElement
```

Comment:
I know this seems obvious to you as to why, but can you be a bit more verbose here? Something like:
"Note: Resize should not reset pan and zoom when disabling an element. This is because we are only resizing the offscreen canvas to deal with the element which was removed, and do not wish to alter the current state of any other currently enabled element"?

Or something :)